sections/generic-best-practices/test-code-with-debugging-in-mind.md

```diff
@@ -0,0 +1,345 @@
+# Keep abstraction low to ease debugging the tests
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+UI tests are made up of a lot of steps and should accomplish three main things, but two of them are somewhat underestimated:
+
+1.  To test a feature _(the obvious one)_
+2.  To help the reader to understand what code does _(usually underestimate)_
```

Comment:
what **the** code does

To ease debuggability

Reply:
Done and pushed, thank you 😊